send-report.sh

```diff
@@ -0,0 +1,209 @@
+#!/bin/bash -e
+
+BUNDLE_TEMPLATE="transaction-bundle-template.json"
+
+usage() {
+    echo "Usage: $0 <report-file> <report-server> <author-ident-sys> <author-ident-val> <proj-ident-sys> <proj-ident-val>"
+    echo "       $0 <report-file> <report-server> <author-ident-sys> <author-ident-val> <proj-ident-sys> <proj-ident-val> [-u <user> -p <password>]"
+    echo "       $0 <report-file> <report-server> <author-ident-sys> <author-ident-val> <proj-ident-sys> <proj-ident-val> [-i <issuer-url> -c <client-id> -s <client-secret>]"
+    exit 1
+}
+
+if [[ $# -lt 6 ]]; then
+    usage
+fi
+
+report_file="$1"
+report_server="$2"
+author_ident_sys="$3"
```

Comment:
I would hard code the fix arguments. Otherwise they could be misspelled via user error.